### DISCUSSION TITLE
Show off your textadept bling!

### DISCUSSION THREAD

**Discussion by benjcal** [orbitalquark/textadept#491] [General]
I was thinking, when I look at other editors pages they show cool color themes and fancy fonts and whatnot. I’m sure that many of you guys have fancy textadept configs, do you wanna show off?

I think this might help not only show off what textadept can do but also give ideas to those of us less imaginative 😅

**Comment by orbitalquark** [comment]
I don't have anything fancy. The screenshots in the manual reflect my setup for the most part, with the exception that I use proportional fonts.

**Comment by paaguti** [comment]
The only ¡fancy' things is that I use Themecreator to create themes when I
get tired of one specific aooearance


On Mon, 11 Dec 2023 at 16:18, orbitalquark ***@***.***> wrote:

> I don't have anything fancy. The screenshots in the manual reflect my
> setup for the most part, with the exception that I use proportional fonts.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/orbitalquark/textadept/discussions/491#discussioncomment-7821173>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAZKU4B2BIIR7A6OLPFAQZLYI4P5ZAVCNFSM6AAAAABANOKWYOVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM3TQMRRGE3TG>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***
> com>
>


-- 
Fragen sind nicht da, um beantwortet zu werden,
Fragen sind da um gestellt zu werden
Georg Kreisler

Headaches with a Juju log:
unit-basic-16: 09:17:36 WARNING juju.worker.uniter.operation we should run
a leader-deposed hook here, but we can't yet
